.github/actions/docker-publish/action.yml

```diff
@@ -34,34 +34,34 @@ runs:
       shell: bash
 
     - name: Log in to Docker Hub
-      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      uses: docker/login-action@f75d088332b07a08afadf6ac53c74509b9453f12
       with:
         username: ${{ inputs.DOCKER_USERNAME }}
         password: ${{ inputs.DOCKER_ACCESS_TOKEN }}
 
     - name: Log in to the Container registry
-      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      uses: docker/login-action@f75d088332b07a08afadf6ac53c74509b9453f12
       with:
         registry: ghcr.io
         username: ${{ github.actor }}
         password: ${{ inputs.GITHUB_TOKEN }}
 
     - name: Extract metadata (tags, labels) for Docker
       id: meta
-      uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+      uses: docker/metadata-action@314ddf6d4a82333bf1bc7630399df41bf68eba09
       with:
         images: |
           trufflesuite/ganache
           ghcr.io/${{ github.repository}}
 
     - name: Set up QEMU
-      uses: docker/setup-qemu-action@0522dcd2bf084920c411162fde334a308be75015
+      uses: docker/setup-qemu-action@38836d31bd1e0f8f29b8c7ad0412da32020f15b3
 
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@91cb32d715c128e5f0ede915cd7e196ab7799b83
+      uses: docker/setup-buildx-action@39a1a82492fd1ad19af19d61b5f748e4cb6cd1af
 
     - name: Build and push Docker image
-      uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+      uses: docker/build-push-action@175d02bffea74695e96b351069ac938b338802f9
       with:
```

Comment:
I guess risk is fairly low, but if I'm interpreting this correctly, these are `master` commits that are _not_ tagged releases. We might want to consider switching to tagged releases at some stage 🤷

Reply:
Thoughts on this perspective? https://blog.natterstefan.me/why-you-should-not-use-the-version-tag-or-branch-name-when-using-github-actions

Reply:
That's a very interesting take, and definitely makes sense. As an addendum, I imagine it would make sense to use the commit hash of a tagged release - that way the author is stating they intend for this revision to be used (and assumedly it is more stable than intermediate commits :shrug:)

Reply:
I did read through all of the commits (ok... I read their titles) since their previous releases and everything seemed innocuous enough.